FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderTests.swift

```diff
@@ -87,23 +56,51 @@ final class ModelDownloaderTests: XCTestCase {
     XCTAssertEqual(modelInfoRetriever.modelInfo?.path, nil)
   }
 
+  /// Unit test to download model info - makes an actual network call.
```

Comment:
We try to separate tests that require secrets like keys or .plist from pure tests that can be run right after the repo checkout to improve dev experience. You can use [FIS](https://github.com/firebase/firebase-ios-sdk/blob/0842ea46153e1926f2f071d1f317d60eae9cba59/FirebaseInstallations.podspec#L67) or [GDT](https://github.com/firebase/firebase-ios-sdk/blob/0842ea46153e1926f2f071d1f317d60eae9cba59/GoogleDataTransport.podspec#L102) as an example. 
It can be addressed in the following PRs.

Reply:
The examples are super helpful! I've added TODOs to break these out into integration tests and create pure unit tests.